.gitlab-ci.yml

```diff
@@ -114,6 +114,10 @@ default:
     - if: $CI_COMMIT_REF_NAME =~ /^[0-9]+$/                         # PRs
 
 .build-refs:                       &build-refs
+  variables:
+    DOCKER_USER:                 "${Docker_Hub_User_Parity}"
+    DOCKER_PASS:                 "${Docker_Hub_Pass_Parity}"
```

Comment:
Now you should use these var names:
https://github.com/paritytech/substrate/blob/master/.gitlab-ci.yml#L174
https://github.com/paritytech/substrate/blob/master/.gitlab-ci.yml#L177
and also please rebase on master.